### PR TITLE
Fractional user rate callout

### PR DESCRIPTION
**Pull request by stb13579**
Addresses the use case where requests per second are desired. Also small language improvements

### CODE REVIEW

**Review by Isammoc — APPROVED** [review]

**Review by slandelle — CHANGES_REQUESTED** [review]
Please also update the code samples to demonstrate fractional values.

**Comment by stb13579** [comment]
> Please also update the code samples to demonstrate fractional values.

The code tabs were not working for me locally, but I think they are OK based on the other examples in the documentation.

File: src/docs/content/reference/current/core/injection/code/InjectionSampleKotlin.kt [hunk 1]
```diff
@@ -39,6 +39,15 @@ setUp(
 )
 //#open-injection
 
+//#fractional-user
+//Injects 1 user every 2 seconds for 60 seconds
+setUp(
+  scn.injectOpen(
+    constantUsersPerSec(0.5).during(60)
+  ).protocols(httpProtocol)
+);
```
Comment (slandelle) [inline src/docs/content/reference/current/core/injection/code/InjectionSampleKotlin.kt:48]: no semi colon in Kotlin

File: src/docs/content/reference/current/core/injection/index.md [hunk 2]
```diff
@@ -58,6 +58,11 @@ The building blocks for open model profile injection are:
 7. `rampUsersPerSec(rate1).to(rate2).during(duration).randomized`: Injects users from starting rate to target rate, defined in users per second, during a given duration. Users will be injected at randomized intervals.
 8. `stressPeakUsers(nbUsers).during(duration)`: Injects a given number of users following a smooth approximation of the [heaviside step function](http://en.wikipedia.org/wiki/Heaviside_step_function) stretched to a given duration.
 
+{{< alert tip >}}
+Fractional user injection rates are allowed. A common usecase is if you want to express the injection in terms of requests per second and not users per second.
```
Comment (slandelle) [inline src/docs/content/reference/current/core/injection/index.md:62]: No found of this. It introduces a confusing between users per sec and requests per sec.
This feature is not directly related to rps.

File: src/docs/content/reference/current/core/injection/code/InjectionSampleKotlin.kt (hunk 1 above)
Comment (slandelle) [reply src/docs/content/reference/current/core/injection/code/InjectionSampleKotlin.kt:48]: Do we want to add more code samples? Or just update the existing one with fractional values? https://github.com/gatling/gatling/blob/364da77b5702689a1cc685e50e1a5bf262ba84ae/src/docs/content/reference/current/core/injection/code/InjectionSampleJava.java#L37-L40

**Review by slandelle — CHANGES_REQUESTED** [review]
